form/app/Models/Product.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Product extends Model
+{
+    use HasFactory;
+    public $timestamps = false;
```

Comment:
@Yashbanu why $timestamps is FALSE.. ?

Reply:
@Yashbanu I just tried with both true and false to see its results , I can change it.